src/test/java/tests/FilterEvalTests.java

```diff
@@ -425,19 +425,39 @@ public void updated() {
         issue.setUpdatedAt(now.minusDays(2));
 
         assertEquals(false, matches("updated:<24", issue));
+        assertEquals(false, matches("updatedByOthers:<24", issue));
+        assertEquals(false, matches("updatedBySelf:<24", issue));
```

Comment:
I added tests to check for `updatedByOthers` and `updatedBySelf` separately with the `IssueMetadata` construction to indicate when the issue was last self updated or last updated by others.